src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Proxy agent chunk requests and analytics data
+tags:
+  - Browser monitoring
+  - Proxy
+metaDescription: "The agent can be configured to fetch its code and send analytics through a proxy server instead of New Relic's endpoints."
+---
+
+From [version 1.240.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, the browser agent officially allows a proxy url to be set when determining either or both of:
+
+1. Where to fetch or lazy-load agent code.
+2. Where to send analytics.
+
+Today, many ad-blockers and security VPNs block client requests to third-party domains including New Relic's. As the number of internet users opting into the use of ad-blockers increase, customers will continue to lose observability on those devices. One solution would be to proxy those requests though an unblocked middleman. For example, `yoursite.com` can send its visitor analytics through its `nrbeacon.yoursite.com` proxy which would not be blocked by anti-adware being that it's considered a first-party (sub)domain.
+
+<Callout variant="important">
+  This feature is currently available for those using the copy-and-paste _or_ NPM installation methods. There is currently no UI or NerdGraph configuration options available. We are continuing to work on improving access to these and other configuration options.
```

Comment:
We should use consistent wording or naming when we call out the copy and paste installation method.

Reply:
Is there a most common phrasing right now?